bindings/pyfiction/include/pyfiction/algorithms/verification/equivalence_checking.hpp

```diff
@@ -60,8 +60,21 @@ inline void equivalence_checking(pybind11::module& m)
     py::class_<fiction::equivalence_checking_stats>(m, "equivalence_checking_stats",
                                                     DOC(fiction_equivalence_checking_stats))
         .def(py::init<>())
+        .def_readonly("eq", &fiction::equivalence_checking_stats::eq, DOC(fiction_equivalence_checking_stats_eq))
+        .def_readonly("tp_spec", &fiction::equivalence_checking_stats::tp_spec,
+                      DOC(fiction_equivalence_checking_stats_tp_spec))
+        .def_readonly("tp_impl", &fiction::equivalence_checking_stats::tp_impl,
+                      DOC(fiction_equivalence_checking_stats_tp_impl))
+        .def_readonly("tp_diff", &fiction::equivalence_checking_stats::tp_diff,
+                      DOC(fiction_equivalence_checking_stats_tp_diff))
         .def_readwrite("counter_example", &fiction::equivalence_checking_stats::counter_example,
```

Comment:
@marcelwa What about `counter_example`?

Reply:
Let's make it `def_readonly` too for consistency